app/loader.cpp

```diff
@@ -308,18 +302,6 @@ void Loader::appStateChanged( Qt::ApplicationState state )
 
   logHelper << "Application changed state to: " << state;
   CoreUtils::log( "Input", msg );
-
```

Comment:
should here be called PositionKit::appStateChanged?

Reply:
Well, I connected it directly from main.cpp [here](https://github.com/lutraconsulting/input/pull/1888/files#diff-c1b709b8a90bef4573ddad8a9b2fbc5d5dbc85e4dc759f95b6d9905931fec24fR450) in order to remove the coupling of loader (which from my point of view should serve only to load a project) and position.

Reply:
ok fair enough. is the original "Loader: appStateChanged" needed at all now?